CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing Guidelines
+
+Thank you for considering contributing to our project! Please take a moment to review the following guidelines.
+
+## Pull Requests
+
+- All contributions to the project must be submitted via pull request.
+- Ensure that your pull request addresses a specific issue or feature request. If none exists, please open a new issue first to discuss the changes you'd like to make.
+- Follow the [GitHub Flow](https://guides.github.com/introduction/flow/) workflow:
+  1. Create a new branch from `main`. The branch name should start with the issue number.
+  2. Make your changes and commit them with descriptive commit messages.
+  3. Submit a pull request to the main repository's `main` branch.
+- Ensure that your code adheres to the project's coding standards and conventions.
+- Include unit tests if applicable and ensure that existing tests pass.
+- Provide a clear and detailed description of your changes in the pull request description.
+
+## Issues
+
+- Feel free to open an issue for bug reports, feature requests, or general feedback.
+- When opening an issue, please provide as much detail as possible, including steps to reproduce for bug reports.
+
+## License
+
+By contributing to this project, you agree that your contributions will be licensed under the project's [LICENSE](LICENSE.txt).
+
+## Releasing
```

Comment:
releasing is not part of contributing

Reply:
Yes good point, what is a better place for this information. I wanted to have it somewhere

Reply:
I would make a RELEASE_GUIDELINES.md